components/image/picker/src/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import ImagePlaceholder from '@schibstedspain/sui-image-placeholder'
+
+class ImagePicker extends Component {
+  constructor (props) {
+    super(props)
+
+    const { images } = props
+    let filteredImages = []
+    let selected = -1
+
+    if (images.length > 0) {
+      filteredImages = images.filter(image => image.src && image.thumb)
+
+      if (filteredImages.length > 0) {
+        selected = filteredImages.findIndex(({ selected }) => selected)
+        selected = (selected === -1) ? 0 : selected
+      }
+    }
+
+    this.state = {
+      selected,
+      images: filteredImages
+    }
+  }
+
+  _getHeadImage () {
+    const { images, selected } = this.state
+    const { emptyImage } = this.props
+
+    const placeHolder = {
```

Comment:
If it's really a constant, IMO you could declare it ONCE at the top of the file.
It's a tradeoff between performance and memory. Up to you.